src/common/snippets/src/op/subgraph.cpp

```diff
@@ -267,7 +275,8 @@ auto Subgraph::constant_input_should_be_inside_body(const std::shared_ptr<ov::No
     return ov::is_type<ov::op::v1::Transpose>(node) ||
            ov::is_type<ov::op::v1::Broadcast>(node) ||
            ov::is_type<ov::op::v3::Broadcast>(node) ||
-           ov::is_type<ov::op::v1::Reshape>(node);
+           ov::is_type<ov::op::v1::Reshape>(node) ||
+           ov::is_type<op::ReduceBase>(node);
```

Comment:
Do we really need to extend `constant_input_should_be_inside_body` list with `ReduceBase`? `ov::snippets::op::ReduceBase` doesn't have constant inputs,

Reply:
applied, thanks Vladislav!